### PR TITLE
new-blogs added

### PR DESCRIPTION
**Pull request by AdarshKumar02291**
Added new blogs from the google docs . The description of image is empty for now will update it . The get price button was taking to a table but we have many tables so it was not working , so for now i have commented it.

### CODE REVIEW

File: api/getBlogs/data/Touroperators.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+    "title": "Your Guide to Finding the Best Tour Operators in Varanasi",
```
Comment (DEVAbhinav) [inline api/getBlogs/data/Touroperators.json]: File name should be Best Tour Operators in Varanasi"

File: api/getBlogs/data/Touroperators.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "metaDescription": "Looking for the perfect tour guide in Varanasi? Discover how to choose the best tour operators for an unforgettable experience in this holy city.",
+    "pageName": "best-tour-operators-in-varanasi",
+    "description": "Essential tips and insights on selecting the ideal tour operator in Varanasi to ensure a memorable journey through this sacred city.",
+    "keywords": ["Varanasi tour operators", "tour guide Varanasi", "Ganga Aarti", "boat rides Ganges", "Varanasi tours"],
+    "images": {
+      "id": "sHWTwOyZ5mflEoXP",
+      "description": "",
+      "file": ["../images/touroperator.png"]
```
Comment (DEVAbhinav) [inline api/getBlogs/data/Touroperators.json]: What is size of image? Is it under 70 KB?

File: api/getBlogs/data/VaranasiAyodhyaAllahbad.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
+{
+    "title": "Embark on a Spiritual Journey: The Ultimate Varanasi, Ayodhya, and Allahabad Tour Package Guide",
```
Comment (DEVAbhinav) [inline api/getBlogs/data/VaranasiAyodhyaAllahbad.json]: File name is Varanasi, Ayodhya, and Allahabad Tour Package Guide. See title and H1 to find page name/route

File: app/public/images/Sighseeing0.png
Comment (DEVAbhinav) [inline app/public/images/Sighseeing0.png:1]: Too much size send via whatsapp  to compress under 80kb stricktly

File: app/public/images/Sighseeing1.png
Comment (DEVAbhinav) [inline app/public/images/Sighseeing1.png:1]: Run size reduction of image for all images in code. For all other images in code do in other PR as a sperate task. but for images included in this PR updated reduced size image

File: app/src/components/navbar/NavbarComponent.jsx [hunk 4]
```diff
@@ -77,6 +77,15 @@ function NavbarComponent() {
                 <Link to="/blogs/Durgamandir" className="font-15 text-brown font-regular dropdown-item">
                   Durga Mata Mandir
                 </Link>
+                <Link to="/blogs/VaranasiItinary" className="font-15 text-brown font-regular dropdown-item">
+                  Varanasi Itinary
```
Comment (DEVAbhinav) [inline app/src/components/navbar/NavbarComponent.jsx]: Hard coding name is bad practice. Try picking it from constants file. Create one if it do not exists.

File: app/src/components/navbar/NavbarComponent.jsx [hunk 5]
```diff
@@ -77,6 +77,15 @@ function NavbarComponent() {
...
+                </Link>
+                <Link to="/blogs/VarnasiAyodhyaAllahbad" className="font-15 text-brown font-regular dropdown-item">
+                  Varansi Ayodhya Allahbad
```
Comment (DEVAbhinav) [inline app/src/components/navbar/NavbarComponent.jsx]: Name should be same as page name. Page name detailed is suggessted earlier.

**Review by DEVAbhinav — COMMENTED** [review]
Please review @dev-1ce

File: api/getBlogs/data/VaranasiAyodhyaAllahbad.json (hunk 3 above)
Comment (dev-1ce) [reply api/getBlogs/data/VaranasiAyodhyaAllahbad.json:2]: 149 KB

File: api/getBlogs/data/Touroperators.json (hunk 2 above)
Comment (AdarshKumar02291) [reply api/getBlogs/data/Touroperators.json]: 149 KB

File: app/public/images/Sighseeing0.png
Comment (AdarshKumar02291) [reply app/public/images/Sighseeing0.png:1]: ok

File: app/public/images/Sighseeing1.png
Comment (AdarshKumar02291) [reply app/public/images/Sighseeing1.png:1]: ok

File: app/src/components/navbar/NavbarComponent.jsx (hunk 4 above)
Comment (AdarshKumar02291) [reply app/src/components/navbar/NavbarComponent.jsx]: ok

File: app/src/components/navbar/NavbarComponent.jsx (hunk 5 above)
Comment (AdarshKumar02291) [reply app/src/components/navbar/NavbarComponent.jsx]: ok

**Comment by AdarshKumar02291** [comment]
Fixed the names of the files . Removed hard coding from the navbar blog and packages options . Images size reduction will be done in a new PR

**Pull request merged**